test/README.md

```diff
@@ -0,0 +1,101 @@
+# End-to-End Tests
+
+The end-to-end tests use Puppeteer and Jest to connect to a running instance of the application and use it through a browser like a normal user would.
```

Comment:
could you please make Puppeteer and Jest into hyperlinks, so it is easy to discover what those projects are really?